core/src/test/java/org/hibernate/ogm/backendtck/associations/order/OpenSourceProject.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.associations.order;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+
+@Entity
+public class OpenSourceProject {
+
+	@Id
+	private String name;
+
+	@OneToMany
+	@JoinTable(name = "OpenProject_contributors")
```

Comment:
Is the JoinTable Needed? I prefer to keep the test classes as simple as poissible

Reply:
I think, in general, it's better to create new models when we introduce new tests. This way we can change the model without the risk of affecting existing tests.